### PR TITLE
Label namespaces with lagoon/project in k8s

### PR DESCRIPTION
**Pull request by smlx**
# Checklist
- [ ] Affected Issues have been mentioned in the Closing issues section
- [ ] Documentation has been written/updated
- [x] PR title is ready for changelog and subsystem label(s) applied

This change adds a label to namespaces created by lagoon to indicate the lagoon project that they belong to. Currently k8s only, not openshift.

This is in support of a larger logging overhaul. Partially adresses #1364.

# Closing issues
n/a

### CODE REVIEW

File: services/kubernetesbuilddeploy/src/index.js [hunk 1]
```diff
@@ -307,7 +307,7 @@ const messageConsumer = async msg => {
   let namespaceStatus = {}
   try {
     const namespacePost = promisify(kubernetes.namespace.post)
-    namespaceStatus = await namespacePost({ body: {"apiVersion":"v1","kind":"Namespace","metadata":{"name":openshiftProject}} })
+    namespaceStatus = await namespacePost({ body: {"apiVersion":"v1","kind":"Namespace","metadata":{"name":openshiftProject, "labels": {"lagoon/project": projectName}}} })
```
Comment (Schnitzel) [inline services/kubernetesbuilddeploy/src/index.js]: we just bought `lagoon.sh` domain, so we can actually use `lagoon.sh` as the label prefix :)
also can we add the environment name in here as well?

File: tests/checks/check-namespace-labels.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+- name: "{{ testname }} - Check if {{project}} namespace is labelled"
```
Comment (Schnitzel) [inline tests/checks/check-namespace-labels.yaml:2]: nice! definitely wanted to implement a test to check something via kubectl 💚

**Review by Schnitzel — CHANGES_REQUESTED** [review]

File: services/kubernetesbuilddeploy/src/index.js (hunk 1 above)
Comment (smlx) [reply services/kubernetesbuilddeploy/src/index.js]: absolutely.. nice domain! :tada:

File: tests/checks/check-namespace-labels.yaml (hunk 2 above)
Comment (smlx) [reply tests/checks/check-namespace-labels.yaml:2]: :+1:

**Comment by smlx** [comment]
@Schnitzel ready for review again, I think your comments have been addressed.

**Pull request merged**